components/api/db/cellery_hub_db.bal

```diff
@@ -32,20 +32,30 @@ type RegistryArtifactImage record {|
     string ARTIFACT_IMAGE_ID;
 |};
 
-public function getOrganization(string orgName) returns json | error {
-    log:printDebug(io:sprintf("Performing data retreival on REGISTRY_ORGANIZATION table, Org name : \'%s\': ", orgName));
-    table<record {}> res = check connection->select(GET_ORG_QUERY, gen:OrgResponse, orgName, loadToMemory = true);
-    if (res.count() == 1) {
-        gen:OrgResponse orgRes = check gen:OrgResponse.convert(res.getNext());
-        json resPayload = check json.convert(orgRes);
-        log:printDebug(io:sprintf("Fetching data for organization \'%s\' from REGISTRY_ORGANIZATION is successful", orgName));
-        res.close();
-        return resPayload;
+public function getOrganization(string orgName, string userId) returns json | error {
+    log:printDebug(io:sprintf("Performing data retreival on REGISTRY_ORGANIZATION table by user \'%s\', Org name : \'%s\': ",
+    userId, orgName));
+    table<record {}> res = check connection->select(GET_ORG_QUERY, gen:OrgResponse, orgName, userId, orgName);
+    int counter = 0;
+    gen:OrgResponse orgRes;
+    foreach var item in res {
+        orgRes = check gen:OrgResponse.convert(item);
+        counter += 1;
+    }   
+    json resPayload = null;  
+    if counter == 1 {
```

Comment:
Since this is a debug log, lets handle the counter == 1, == 0 and > 1 conditions separately.